### ISSUE TITLE
Bug report: logCustom not use TalkerSettings getPenByLogKey color in TalkerScreen

### ISSUE THREAD

**Issue by a1573595** [Frezyx/talker#288]
**Describe the bug**
TalkerSettings defines the default color of the log level. When TalkerLog specifies a level but does not specify a color, the color settings of TalkerSettings should be used.

**To Reproduce**
1. Extend the TalkerLog class and define the log level.
2. use talker.logCustom print log.
3. Open TalkerScreen see the log.

**Expected behavior**
Default uses the color settings of TalkerSettings.

**Screenshots**
Android Studio console works well.
![截圖 2024-11-28 下午3 07 52](https://github.com/user-attachments/assets/9c180aff-f8e5-4fb3-a3b9-426e20f762c6)

TalkerScreen has no color.
<img width="326" alt="截圖 2024-11-28 下午4 00 00" src="https://github.com/user-attachments/assets/ff42c19b-849e-45b3-8da4-baef1b5e1493">

**Desktop (please complete the following information):**
 - OS: MacOS 15.1.1（24B91）

**Smartphone (please complete the following information):**
 - Device: Pixel 8a
 - OS: Android 34
 - Version: talker_flutter: ^4.5.1

**Comment by Frezyx** [comment]
Hello @a1573595 ! 
Thank you for creating this issue. I will check this bug in the near future

**Comment by Frezyx** [comment]
Can you provide an code example of your own custom log ?

**Comment by a1573595** [comment]
> Can you provide an code example of your own custom log ?

Sample code：
```dart
void main() {
  final talker = Talker();

  talker.logCustom(TalkerLog("debug", logLevel: LogLevel.debug));
  talker.logCustom(TalkerLog("error", logLevel: LogLevel.error));

  runApp(MaterialApp(
    home: TalkerScreen(talker: talker),
  ));
}
```

**Comment by Frezyx** [comment]
@a1573595 please check this PR https://github.com/Frezyx/talker/pull/289
This is a new feature for TalkerScreen, which was not available before

**Comment by Frezyx** [comment]
@a1573595 Fix now available in [4.5.2](https://pub.dev/packages/talker_flutter) package version
